src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -143,6 +145,21 @@ private function replaceByShortType(Tokens $tokens, TypeAnalysis $type, array $u
             }
 
             if (!str_starts_with($typeName, '\\')) {
+                if ('' === $namespaceName) {
+                    // if we are in the global namespace and the type is not imported enforce the leading '\'
+                    // to unify CS with namespaced files
+                    $typeNameLower = strtolower($typeName);
+                    if (!isset($uses[$typeNameLower])) {
+                        foreach ($uses as $useShortName) {
+                            if (strtolower($useShortName) === $typeNameLower) {
+                                continue 2;
+                            }
+                        }
+
+                        $tokens->overrideRange($startIndex, $endIndex, $this->namespacedStringToTokens($typeName, true));
```

Comment:
The leading backslash behavior should be configurable, and available as an opt-in to keep current behavior. We just can't change current behavior 🙂.

Reply:
done as opt-out for the reasons mentioned in the PR description